generators/cypress/templates/src/test/javascript/cypress/integration/account/reset_password_page_spec.ts.ejs

```diff
@@ -0,0 +1,40 @@
+import {
+    usernameLoginSelector,
+    forgetYourPasswordSelector,
+    emailResetPasswordSelector,
+    newResetPasswordSelector,
+    confirmResetPasswordSelector,
+    submitInitResetPasswordSelector,
+    submitFinishResetPasswordSelector
+   } from '../../support/commands';
+
+  describe('forgot your password', () => {
+    beforeEach(() => {
+      cy.server();
+      cy.route('POST', '/api/account/reset-password/init').as('initResetPassword');
+      cy.route('POST', '/api/account/reset-password/finish').as('finishResetPassword');
+    });
+
+    before(() => {
+      cy.visit('');
+      cy.clickOnLoginItem();
+      cy.get(usernameLoginSelector).type('admin');
+      cy.get(forgetYourPasswordSelector).click();
+    });
+
+    it('should be able to init reset password', () => {
+      cy.get(emailResetPasswordSelector).clear().type('user@gmail.com');
+      cy.get(submitInitResetPasswordSelector).click({ force: true });
+      cy.wait('@initResetPassword').its('status').should('equal', 200);
+    });
+
+    it('should be able to finish reset password', () => {
+      /** TODO
```

Comment:
TODO ? remove or implement

Reply:
We will remove it because for the moment it is not used.